test/net/sourceforge/kolmafia/maximizer/MaximizerTest.java

```diff
@@ -110,18 +110,20 @@ public void startingMinKeywordPassesMaximizationIfHit() {
   @Test
   public void clownosityTriesClownEquipment() {
     canUse("clown wig");
-    assertFalse(maximize("clownosity"));
+    assertFalse(maximize("clownosity -tie"));
```

Comment:
What is different because of -tie?  Since tests are "no cost" why not add an additional test if clownosity and clownosity -t do different things?

Reply:
I realised that the test wasn't actually testing anything. The only items in inventory were clown items, and the maximizer tries to equip any items it finds according to the tiebreaker, so the test passes if you maximize "-club" or "current" or "empty" or any other valid string that doesn't actually do anything.

To test without the tiebreaker, I'd have to add items the tiebreaker considered "better" than the clown wig, then maximize clownosity and confirm the wig was preferred. I thought it was easier to just disable it.

Reply:
The fact that you are changing an existing test causes concern.  What this says to me is that your changes broke an existing test so you change the test.   There are cases where the test was flawed and the test needs to be fixed.  But I wonder if this is just leakage and would not be necessary if your new tests just cleaned up after themselves.

Reply:
It's not leakage, the test was flawed. It still passes without the `-tie` change. But this pass doesn't guarantee that the clownosity keyword actually works. The test existed, but failed to test what it was supposed to test.

For example, here is one of the tests as it originally was:
```
  @Test
  public void clownositySucceedsWithEnoughEquipment() {
    canUse("clown wig");
    canUse("polka-dot bow tie");
    assertTrue(maximize("clownosity"));
    recommendedSlotIs(EquipmentManager.HAT, "clown wig");
    recommendedSlotIs(EquipmentManager.ACCESSORY1, "polka-dot bow tie");
  }
```

however this "version" of the test also passes:
```
  @Test
  public void clownositySucceedsWithEnoughEquipment() {
    canUse("clown wig");
    canUse("polka-dot bow tie");
    assertTrue(maximize("current"));
    recommendedSlotIs(EquipmentManager.HAT, "clown wig");
    recommendedSlotIs(EquipmentManager.ACCESSORY1, "polka-dot bow tie");
  }
```
(I have swapped out `maximize("clownosity")` for `maximize("current")`). I intended to test that the clownosity keyword favoured clowny equipment. However, what I actually tested was that maximizing favours some equipment over no equipment.

Happy to revert the change if it would make you happier :)

Reply:
Honestly this explanation makes sense to me, @jaadams5. I vote to keep the `-tie`.

Reply:
Nope (to reverting).  If the test under question was flawed then the change isn't hurting.